src/midiToNotes.js

```diff
@@ -251,7 +256,7 @@ const MidiToNotes = (function () {
     let currTime = 0;
 
     for (const event of sortedMidiEvents) {
-      let adjustedDeltaTime = event.deltaTime * currTempo / baseTempo;
+      let adjustedDeltaTime = event.deltaTime * (currTempo / baseTempo);
```

Comment:
What effect does this have?

Reply:
yeah this was an attempt to mitigate float inaccuracy before I found the actual fix, just forgot to remove it. Whoops